release_bot/init_repo.py

```diff
@@ -101,12 +101,20 @@ def create_conf(self, silent):
             print("""For details on how to get github token checkout
     'https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/'""")
             self.conf['github_token'] = input('Please enter your valid github token:')
-            refresh_interval = input("""In how many seconds would you like the
+            check_valid_interval = True
+            while check_valid_interval:
+                refresh_interval = input("""In how many seconds would you like the
```

Comment:
You can check whether the entered value has correct format (i.e. is digit) directly in the `while`:

```python
refresh_interval = "dummy"
while not (refresh_interval.isdigit() or refresh_interval == ""): 
    refresh_interval = input("In how many seconds ...")

# the rest stays the same, i.e.:
if refresh_interval:
    self.conf['refresh_interval'] = int(refresh_interval)
else:
    self.conf['refresh_interval'] = 180
```

Reply:
Simple, and easy. I'll make the changes. :D